src/web/app/components/sessions-table/send-reminders-to-respondents-modal/send-reminders-to-respondents-modal.component.ts

```diff
@@ -64,10 +67,17 @@ export class SendRemindersToRespondentsModalComponent {
     });
   }
 
+  /**
+   * Changes selection state for sending a copy to requesting instructor.
+   */
+  changeSelectionStatusForSendingCopyToInstructorHandler(shouldSelect: boolean): void {
+    this.isSendingCopyToInstructor = shouldSelect;
+  }
+
   /**
    * Collates a list of selected students with selected status.
    */
-  collateRespondentsToSendHandler(): (StudentListInfoTableRowModel | InstructorListInfoTableRowModel)[] {
+  collateRespondentsToSend(): (StudentListInfoTableRowModel | InstructorListInfoTableRowModel)[] {
```

Comment:
I believe this function can now be private. Let's also move it below `collateReminderResponseHandler` so we can group the handlers together.